vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -190,9 +193,129 @@ - (void)dealloc
   delete rewardsDatabase;
 }
 
+- (void)initializeLedgerService:(BOOL)executeMigrateScript
+{
+  if (self.initialized || self.initializing) {
+    return;
+  }
+  self.initializing = YES;
+  
+  BLOG(ledger::LogLevel::LOG_DEBUG) << "DB: Migrate from CoreData? " << (executeMigrateScript ? "YES" : "NO") << std::endl;
+  ledger->Initialize(executeMigrateScript, ^(ledger::Result result){
+    self.initialized = result == ledger::Result::LEDGER_OK;
+    self.initializing = NO;
```

Comment:
```
self.initialized = result == ledger::Result::LEDGER_OK;
self.initializing = NO;
```

Is the first line necessary or the second one?

Reply:
They're 2 different properties. The first being whether or not the ledger service has initialized (result == OK), and the second is saying it is no longer in the middle of initialization.

Reply:
Ahhh I misread lol. I saw both as `initialized`